app/models/host/managed.rb

```diff
@@ -129,7 +129,6 @@ class Jail < ::Safemode::Jail
   alias_attribute :os, :operatingsystem
   alias_attribute :arch, :architecture
 
-  validates :environment_id,  :presence => true
```

Comment:
@jlsherrill ^ that seems safer than completely disabling it (at least until we extract Puppet to a plugin)